### PR TITLE
Generate: skip left-padding tests on old models

### PR DESCRIPTION
**Pull request by gante**
# What does this PR do?

We have a few left-padding tests polluting our daily CI -- these are from old models, where it is not worth committing >1hr per model to add support for left padding.

This PR skips the test in those models, so we can focus our energy where it matters :)

### CODE REVIEW

File: tests/models/bart/test_modeling_bart.py [hunk 1]
```diff
@@ -1516,3 +1516,7 @@ def test_retain_grad_hidden_states_attentions(self):
 
     def test_save_load_fast_init_from_base(self):
         pass
+
+    @unittest.skip("The model doesn't support left padding")  # and it's not used enough to be worth fixing :)
```
Comment (gante) [inline tests/models/bart/test_modeling_bart.py:1520]: Note that this is not Bart, but the BartDecoder used as a stand-alone model. 13 out of the 16 skips below follow this pattern, a decoder used as stand-alone

**Review by ydshieh — APPROVED** [review]
Thank you! I am more than happy to see a cleaner daily CI report + You are  the king of generation --> 200% approval for this PR from my side.

**Comment by ydshieh** [comment]
@gante it all passes now 😄

**Review by amyeroberts — APPROVED** [review]
Thanks for adding!

**Pull request merged**